### DISCUSSION TITLE
Containerized AqualinkD

### DISCUSSION THREAD

**Discussion by chrisdaun** [aqualinkd/AqualinkD#284] [General]
@sfeakes, ever consider releasing a docker image of aqualinkd?

Thanks,
Chris

**Comment by sfeakes** [comment]
There is a version AqualinkD-docker.  But it’s designed to be built yourself, I’m currently working on an update to take advantage of new features in AqualinkD and that will be a traditional container.

**Reply by chrisdaun** [reply]
Beautiful!
Wanted to say thanks for this wonderful project, you've clearly put a ton of work into it.
It was such a fun adventure getting my setup up and running :)
